### PR TITLE
cmd/stat: Add feature stat object

### PR DESCRIPTION
**Pull request by Prnyself**

### CODE REVIEW

File: task/object_stat.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+func (t *ObjectStatTask) run() {
+	om, err := t.GetStorage().HeadObject(t.GetKey())
+	if err != nil {
+		panic(err)
+	}
+	oriOm := t.GetObjectMeta()
```
Comment (Xuanwo) [inline task/object_stat.go]: Why not use `t.SetObjectMeta(om)`?

File: task/stat_test.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+)
+
+func TestNewStatTask(t *testing.T) {
+	cases := []struct {
+		input            string
+		expectedTodoFunc types.TodoFunc
+		expectErr        error
+	}{
+		{"qs://test-bucket/obj", NewObjectStatTask, nil},
+		{"qs://test-bucket/obj/", NewObjectStatTask, nil},
```
Comment (Xuanwo) [inline task/stat_test.go]: `obj/` end with `/` should be treated as a pseudo dir in remote.

File: task/tasks.json [hunk 3]
```diff
@@ -235,6 +235,29 @@
...
     ]
   },
+  "ObjectStat": {
+    "type": "required",
+    "path": "object_stat",
+    "description": "stat a remote object by request headObject",
+    "inherited_value": [
+      "Format",
+      "Key",
+      "ObjectMeta",
```
Comment (Xuanwo) [inline task/tasks.json]: `ObjectMeta` is a runtime value for `ObjectStat`, because it will be updated while runtime.

File: cmd/qsctl/stat.go [hunk 4]
```diff
@@ -19,13 +30,68 @@ var StatCommand = &cobra.Command{
...
+		if err = stor.SetupBucket(bucketName, ""); err != nil {
+			return
+		}
+	})
+
+	t.Run()
+	t.Wait()
+
+	// if format string was set, print result as format string
+	if t.GetFormat() != "" {
```
Comment (Xuanwo) [inline cmd/qsctl/stat.go]: If format only used in command line output, we don't need to add a type for it.

File: task/stat_test.go (hunk 2 above)
Comment (Prnyself) [reply task/stat_test.go]: But there is no difference between stating an object and a pseudo dir, both of them  got the same behavior.

Comment (Xuanwo) [reply task/stat_test.go]: Well, I think a NewPseudoDirStatTask is more unambiguous, let's leave a comment here and return error for this saturation.

Comment (Prnyself) [reply task/stat_test.go]: OK，I'll make another commit to handle this.

**Pull request merged**